defi/l2/adapters/ada.ts

```diff
@@ -1,7 +1,7 @@
-import fetch from "node-fetch";
+import { cachedFetch } from "@defillama/sdk/build/util/cache";
```

Comment:
hmm, no. Lets not use this path `@defillama/sdk/build/...` in the future, once the sdk is published, you can get it via `sdk.cache.cachedFetch`

Reply:
Same for the runInPromisePool in the next file